nextcloudappstore/api/v1/release/changes.xsd

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           elementFormDefault="qualified" attributeFormDefault="unqualified">
+    <xs:element name="release">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="changelog" maxOccurs="1" minOccurs="1">
+                    <xs:complexType>
+                        <xs:attribute name="href" type="xs:anyURI" use="required"  />
+                    </xs:complexType>
+                </xs:element>
+                <xs:element name="whatsNew" minOccurs="1" maxOccurs="unbounded">
+                    <xs:complexType>
+                        <xs:sequence>
+                            <xs:element name="regular" type="whatsNewInfo" minOccurs="1" maxOccurs="1" />
```

Comment:
What if there are only admin changes? like "ui for configuring additional mount type"

Reply:
Yeah, we should consider that. Might make sense for the server, too, as the maintenance releases often don't have much to brag about, but subtle bug fixes.